test/images/apparmor-loader/loader.go

```diff
@@ -44,6 +44,7 @@ const (
 )
 
 func main() {
+	klog.InitFlags(nil)
```

Comment:
i believe the latest version of klog no longer needs this (k/k possibly hasn't moved to it yet).
but LGTM

/lgtm
/ok-to-test